doc-botonic/website/sidebars.json

```diff
@@ -0,0 +1,74 @@
+{
+  "docs": {
+    "Getting Started": [
+	"welcome",
+	"getting-started",
+	"create-convapp",
+	"project",
+	"supported-platforms"
+	],
+	
+	"Templates": [
+	"templates/templates",
+	"templates/template-custom-webchat",
+	"templates/childs",
+	"templates/template-dynamic-carousel",
+	"templates/template-human-handoff",
+	"templates/template-nlu"
+	],
+	
+	"Components": [
+	"components/components",
+	"components/co-text",
+	"components/co-buttons",
```

Comment:
would we better to have only `components/buttons` instead of `components/co-buttons`, as buttons already belong to this subpath url. If so, remember to modify all "co-" references

Reply:
Path renamed and updated.